_utils/path_firefox.ini

```diff
@@ -0,0 +1,17 @@
+[darwin]
+PATH_FIREFOX_GR="/Applications/Firefox/Contents/MacOS"
+PATH_FIREFOX_NN="/Applications/FirefoxNightly.app/Contents/MacOS"
+PATH_FIREFOX_DE="/Applications/FirefoxDeveloperEdition/Contents/MacOS"
```

Comment:
Not sure about these 3 paths. I copied them from another script, but I think they should probably have the "firefox" or "firefox-bin" at the end (to be more consistent w/ the Cygwin paths below.